src/CliMenuBuilder.php

```diff
@@ -280,6 +280,37 @@ public function setTitleSeparator(string $separator) : self
         return $this;
     }
 
+    public function setBorder(
+        int $topWidth,
+        $rightWidth = null,
+        $bottomWidth = null,
+        $leftWidth = null,
+        string $colour = null
+    ) : self {
+        if (!is_int($rightWidth)) {
+            $colour = $rightWidth;
+            $rightWidth = $bottomWidth = $leftWidth = $topWidth;
+        } elseif (!is_int($bottomWidth)) {
+            $colour = $bottomWidth;
+            $bottomWidth = $topWidth;
+            $leftWidth = $rightWidth;
+        } elseif (!is_int($leftWidth)) {
+            $colour = $leftWidth;
+            $leftWidth = $rightWidth;
+        }
+
+        $this->style['borderTopWidth'] = $topWidth;
+        $this->style['borderRightWidth'] = $rightWidth;
+        $this->style['borderBottomWidth'] = $bottomWidth;
+        $this->style['borderLeftWidth'] = $leftWidth;
+
+        if (is_string($colour)) {
```

Comment:
Should we throw an exception if colour is not null and not a string? `validateColour` will check the colour later but not if someone does something weird like pass an object to it.

Reply:
Done.